services/metadactyl-clj/src/metadactyl/routes/apps.clj

```diff
@@ -102,79 +102,79 @@
         :summary "Update an App"
         :description "This service updates a single-step App in the database, as long as the App has not
         been submitted for public use."
-        (service/trap uri apps/update-app current-user (assoc body :id app-id)))
+        (ok (apps/update-app current-user (assoc body :id app-id))))
 
-  (POST* "/:app-id/copy" [:as {uri :uri}]
+  (POST* "/:app-id/copy" []
          :path-params [app-id :- AppIdPathParam]
          :query [params SecuredQueryParamsRequired]
          :return App
          :summary "Make a Copy of an App Available for Editing"
          :description "This service can be used to make a copy of an App in the user's workspace."
-         (service/trap uri apps/copy-app current-user app-id))
+         (ok (apps/copy-app current-user app-id)))
 
-  (GET* "/:app-id/details" [:as {uri :uri}]
+  (GET* "/:app-id/details" []
         :path-params [app-id :- AppIdJobViewPathParam]
         :query [params SecuredQueryParams]
         :return AppDetails
         :summary "Get App Details"
         :description "This service is used by the DE to obtain high-level details about a single App"
-        (service/coerced-trap uri AppDetails apps/get-app-details current-user app-id))
+        (service/coerced-trap nil AppDetails apps/get-app-details current-user app-id))
```

Comment:
The `uri` was added here because the old `trap` function expected an `action` parameter, which used to be logged. Since we're logging requests with their `uri` elsewhere now, I don't think this `trap` function needs the `uri` anymore.